library/core/src/iter/traits/collect.rs

```diff
@@ -346,7 +346,6 @@ pub trait IntoIterator {
     fn into_iter(self) -> Self::IntoIter;
 }
 
-#[rustc_const_unstable(feature = "const_intoiterator_identity", issue = "90603")]
```

Comment:
I found two remaining `rustc_const_unstable` attributes on `impl` blocks, I think those should all have been removed with the const trait purge and were probably just forgotten.